Update group-work.md (#57)
coderefinery/github-without-command-line: group-work.md

## Organizational permissions

- Organizations have **owners** and **members**.

> - Owners can add more than members and delete repositories.

Reviewer: ```suggestion
- Owners can add additional members and delete repositories.
```
